JS/Lesson22/tugba/script.js

```diff
@@ -0,0 +1,8 @@
+let acc = document.getElementsByClassName("accordion");
+
+for (let i = 0; i < acc.length; i++) {
+    acc[i].onclick = function () {
+        let panel = this.nextElementSibling;
```

Comment:
You don't change the actual value of the `panel` so you should be able to use `const` here.